axis.go

```diff
@@ -200,7 +202,7 @@ func (a *horizontalAxis) size() (h vg.Length) {
 		h -= a.Label.Font.Extents().Descent
 		h += a.Label.Height(a.Label.Text)
 	}
-	if marks := a.Tick.Marker.Ticks(a.Min, a.Max); len(marks) > 0 {
+	if marks := a.Tick.Marker.Ticks(a.Min, a.Max, nil); len(marks) > 0 {
```

Comment:
ok. so how would I use my own format here?

shouldn't `Axis` grow another field that could be passed there?
something like:

```go
type Axis struct {
   // ...
   Tick struct {
      // ...
      // Marker returns the tick marks.  Any tick marks
      // returned by the Marker function that are not in
      // range of the axis are not drawn.
      Marker Ticker

      // <doc for Format>
      Format func(v float64, prec int) string
   }
}
```

and then:

```go
if marks := a.Tick.Marker.Ticks(a.Min, a.Max, a.Tick.Format); len(marks) > 0 {
```